.github/workflows/integration_cli.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         override: true
         toolchain: 1.73.0
 
+    - name: Use Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
```

Comment:
What is this for? I've deactivated the CLI tests (for now) just wonder if we need to add python the the cirlceci config now..?